components/dashboard/src/projects/Project.tsx

```diff
@@ -130,8 +105,12 @@ export default function () {
         history.push(`/${team?.slug}/${projectName}/${pb.id}`);
     }
 
+    const formatDate = (date: string | undefined) => {
+        return date ? moment(date).fromNow() : "";
+    }
+
     return <>
-        <Header title={project?.name || ""} subtitle={toRemoteURL(project?.cloneUrl || "")} />
+        <Header title={"Overview"} subtitle={<h2>View all active branches for <a className="text-gray-400 hover:text-gray-600" href={project?.cloneUrl}>{project?.name}</a>.</h2>} />
```

Comment:
Nit: I found this "Overview" page a bit confusing -- both its Tab and it's title say just "Overview", and I was unclear on what that even meant. Only by reading the (small, lightgrey) description did I understand "aha, it's the branches!"

Q: Why not call this page "Branches"? Or "Branches Overview"?

Reply:
I think _Branches Overview_ is quite verbose for a page title. Alternatively, we can revert the title back to the project name. What do you think? See also https://github.com/gitpod-io/gitpod/pull/4970#discussion_r677536394.

Reply:
I think the reasons from https://github.com/gitpod-io/gitpod/pull/4876#discussion_r676619019 still apply. Call it `Overview` or `Active Branches` but it would be great if users would immediately get what the list contains, especially that have some views with similar looking lists.

Reply:
Agree, that's one of the reasons we renamed the subtitle here to include specifically _active branches_, which could be also rephrased for accuracy.

```suggestion
        <Header title={"Overview"} subtitle={<h2>View recent active branches for <a className="text-gray-400 hover:text-gray-600" href={project?.cloneUrl}>{project?.name}</a>.</h2>} />
```

🍋 🍋 🍋 🍋 

For the title, the intent was to expand this page with more than recent branches and have separate tabs for Branches, PRs, etc, hence the _Overview_ tab title, but we could be more explicit for now. I'd vote for a) using a single word term like _Overview_ or _Branches_ and b) keeping the tab title as _Overview_. Feel free to go with the one that feels better since this is a **two-way door decision** we can easily update later. 🚪

Reply:
Then I'd also vote for "Branches" 😁 (and we can later rename it "Overview" when we add more content to it)

Reply:
![5hzet3](https://user-images.githubusercontent.com/120486/127515077-a6219ce1-8d21-48be-a6af-c92a439df2b0.jpg)